src/submission/submission-service.ts

```diff
@@ -852,10 +852,10 @@ export namespace operations {
     );
   }
 
-  const applyExceptionIfExists = (
+  const checkExceptionExists = (
```

Comment:
This could use a docstring comment. I think the method returns true if the exception value is permissible because of an exception registered for this program?